key.core/src/main/java/de/uka/ilkd/key/util/MiscTools.java

```diff
@@ -781,7 +764,7 @@ public static URI extractURI(DataLocation loc) {
                 return new URI("urn", loc.toString(), null);
             }
         } catch (URISyntaxException | IOException e) {
-            e.printStackTrace();
+
```

Comment:
Please do not drop an exception w/o logging (even if it is only at a low visibility level). Perhaps move the IAE from below and use e as cause.

Reply:
Woops, I think I meant to throw the exception from two lines below here. Fixed in a follow up commit.